firefox_puppeteer/api/software_update.py

```diff
@@ -0,0 +1,499 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import re
+
+import mozinfo
+
+from ..base import BaseLib
+from .appinfo import AppInfo
+from .prefs import Preferences
+
+
+class ActiveUpdate(BaseLib):
+
+    def __getattr__(self, attr):
+        value = self.marionette.execute_script("""
+          let ums = Cc['@mozilla.org/updates/update-manager;1']
```

Comment:
You still have `Cc` in your patch which would also break us. Please also use `Components.classes` everywhere.